mlrun/config.py

```diff
@@ -234,7 +234,10 @@
                 "model_endpoint_creation": "600",
                 "model_endpoint_tsdb_leftovers": "900",
             },
-            "runtimes": {"dask": "600"},
+            "runtimes": {
+                "dask": "600",
+                "dask_cluster_start": "300",
```

Comment:
Is 5 minutes a reasonable timeout for dask functions?
Can you check for average start time (without errors) and what does it depend on?